bin/sqlToLog

```diff
@@ -124,16 +128,18 @@ ORDER BY visit %s
                     continue
                 if re.search(r"^(COMMENT\s+|END$)", card):
                     continue
-                
+
                 mat = re.search(r"^([^\s]+)\s*=\s*([^\s]+)", card)
                 if mat:
                     k, v = mat.groups()
                     if v in ("T", "F"):
                         v = (v == "T")
                     else:
-                        try: v = int(v)
+                        try:
+                            v = int(v)
                         except:
-                            try: v = float(v)
+                            try:
+                                v = float(v)
                             except:
                                 v = v[1:-1].rstrip()
 
```

Comment:
It's not magic!   That regex splits `key = value`  (`k` and `v` in the code), then does sensible things if v is a logical, int, or float.